### PR TITLE
Feature/login panel

### LINKED ISSUE #179: Add login page

### LINKED ISSUE #204: Send calculation request from frontend to backend
Send calculation request (currently `/sh/demo` route) to the backend. 
Keep in mind that this operation currently in the backend is blocking.

The exact backend location (IP) should be stored on the layer of deployment, preferably in an env variable.

We should use Run tab, add "Start simulation" button and some "progress bar" (spinner?).

Finally we display string returned by backend (respectively if this is a result of simulation or not)

### PR DESCRIPTION
**Pull request by Derstilon**
closes #179 
closes #204 
Added simple UI for login and simulation running

### CODE REVIEW

**Review by grzanka — APPROVED** [review]
Looks good

**Comment by Derstilon** [comment]
> Looks good
@grzanka 
image
Can you apply this?

**Comment by grzanka** [comment]
> > Looks good
> 
> @grzanka image Can you apply this?

Before committing, can you check it ?
It looks that deepsource doesn't like console (prefers modal or some other way of communication):
image

**Comment by Derstilon** [comment]
> > > Looks good
> > 
> > 
> > @grzanka image Can you apply this?
> 
> Before committing, can you check it ? It looks that deepsource doesn't like console (prefers modal or some other way of communication): image

Hmmm. This is all deepsource wanted to fix?

**Comment by grzanka** [comment]
> > > > Looks good
> > > 
> > > 
> > > @grzanka image Can you apply this?
> > 
> > 
> > Before committing, can you check it ? It looks that deepsource doesn't like console (prefers modal or some other way of communication): image
> 
> Hmmm. This is all deepsource wanted to fix?

Yes. This is why I asked you to cross-check before commiting

File: src/util/Config.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export const BACKEND_URL = process.env.REACT_APP_BACKEND_URL ?? 'https://617d3fe41eadc50017136443.mockapi.io';
```
Comment (grzanka) [inline src/util/Config.ts:1]: What is `mockapi.io` ?

File: src/WrapperApp/components/SimulationPanel.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+export default function SimulationPanel(props: SimulationPanelProps) {
+    const { editorRef } = useStore();
+    const [isInProgress, setInProgress] = useState<boolean>(false);
+
+    const sendRequest = () => {
+        setInProgress(true);
+        ky.post(`${BACKEND_URL}/sh/demo`, { json: editorRef.current?.toJSON() })
+            .json()
+            .then((response) => {
+                console.log(response);
```
Comment (grzanka) [inline src/WrapperApp/components/SimulationPanel.tsx:22]: Can't we use some modal popup for debugging purposes here ?

Comment (ostatni5) [reply src/WrapperApp/components/SimulationPanel.tsx]: Yes we can.

File: src/util/Config.ts (hunk 1 above)
Comment (ostatni5) [reply src/util/Config.ts]: Free page where you can define simple rest api.

**Review by ostatni5 — APPROVED** [review]
obraz

**Pull request merged**